plugin/storage/grpc/shared/grpc_handler.go

```diff
@@ -151,7 +151,10 @@ func (s *GRPCHandler) Close(context.Context, *storage_v1.CloseWriterRequest) (*s
 
 // GetTrace takes a traceID and streams a Trace associated with that traceID
 func (s *GRPCHandler) GetTrace(r *storage_v1.GetTraceRequest, stream storage_v1.SpanReaderPlugin_GetTraceServer) error {
-	trace, err := s.impl.SpanReader().GetTrace(stream.Context(), r.TraceID)
+	query := spanstore.GetTraceParameters{
+		TraceID: r.TraceID,
```

Comment:
You are doing the PRs in the wrong order. I recommend first starting with upgrading grpc storage API, then here you'd be able to assign time stamps right away, instead of leaving unintentional bug.

Reply:
Got your point, let's try this first: https://github.com/jaegertracing/jaeger/pull/6244